jvb/src/main/java/org/jitsi/videobridge/xmpp/MediaSourceFactory.java

```diff
@@ -774,13 +801,17 @@ private static MediaSourceDesc createSource(
         // not optional (there's no good reason for that). Then the error will be thrown automatically by Kotlin.
         if (MultiStreamConfig.config.getEnabled())
         {
-            if (primarySsrcs.name == null)
+            if (owner == null)
             {
                 throw new IllegalArgumentException("The 'name' is missing in the source description");
```

Comment:
"owner" and "name" are the wrong way around in the log messages

Reply:
🤦‍♂️